server/controllers/lists/by_id.js

```diff
@@ -0,0 +1,31 @@
+const { byIdsWithSelections } = require('controllers/lists/lib/lists')
```

Comment:
it has some minor advantages of not passing the param `withSelection`, to not warn on `checkNotFoundList`. But the main purpose is to allow Selection pagination, otherwise `by-ids` needs to be paginated...